wagtail/images/models.py

```diff
@@ -403,6 +408,22 @@ def get_rendition_model(cls):
         """Get the Rendition model for this Image model"""
         return cls.renditions.rel.related_model
 
+    def _get_prefetched_renditions(self) -> Union[Iterable["AbstractRendition"], None]:
+        if "renditions" in getattr(self, "_prefetched_objects_cache", {}):
+            return self.renditions.all()
+        return getattr(self, "prefetched_renditions", None)
```

Comment:
Suggestion: To be safer on types, should this return `[]` in the case there are no prefetched renditions? I don't know how necessary it is to distinguish between "There are no renditions prefetched" and "No one has tried to prefetch any renditions"?

Reply:
Thanks @RealOrangeOne. In this case, the difference is quite important to the calling code. Essentially, we need to know the difference between "Renditions were prefetched, but the result was empty" and "Renditions were not prefetched".